saleor/static/js/components/categoryPage/CategoryPage.js

```diff
@@ -106,7 +129,17 @@ class CategoryPage extends Component {
               </ul>
             </div>
             <div className="col-md-4">
-              <SortBy sortedValue={variables.sortBy} setSorting={this.setSorting} />
+              <div className="row">
+                <div className="col-6 col-md-2 col-lg-6 filters-menu">
+                  <span className="filters-menu-label hidden-sm-up" onClick={() => this.toggleMenu(filtersMenu)}>Filters</span>
+                  {variables.attributesFilter.length || variables.minPrice || variables.maxPrice ? (
+                    <span className="filters-menu-icon hidden-sm-up"></span>
+                  ) : (null)}
```

Comment:
This will break if `someCondition || otherCondition` returns something falsy like `0`. React will then render the left side or raise an error when it does not know how to do it.